### DISCUSSION TITLE
可以适配115吗？

### DISCUSSION THREAD

**Discussion by 5536086** [AlistGo/alist#357] [Q&A]
有115挂载需求，能否适配115呢？

**Comment by xhofe** [comment]
普通用户不仅速度慢，且下载需要携带cookie，无适配计划。

**Reply by yangchris4** [reply]
> 普通用户不仅速度慢，且下载需要携带cookie，无适配计划。

求适配比特求哇 大佬

**Reply by We1eVen** [reply]
虽然支不支持是大佬说了算，但是为什么夸克就可以呢

**Reply by tyokantei** [reply]
> 普通用户不仅速度慢，且下载需要携带cookie，无适配计划。

大佬考虑下吧，其实115会员的速度还是可以的，不需要下载，只要能实现webdav播放，用alist调用就行，扫码登录

**Comment by jonntd** [comment]
理论上可以你不怕被封号么

We1eVen ***@***.***> 于2022年6月15日周三 00:00写道：

**Comment by jonntd** [comment]
多ip封号

tyokantei ***@***.***> 于2022年7月21日周四 00:37写道：
